assets/controller.yaml

```diff
@@ -268,49 +268,50 @@ spec:
           volumeMounts:
           - mountPath: /etc/tls/private
             name: metrics-serving-cert
-        - name: csi-snapshotter
-          image: ${SNAPSHOTTER_IMAGE}
-          imagePullPolicy: IfNotPresent
-          args:
-            - --csi-address=$(ADDRESS)
-            - --metrics-address=localhost:8215
-            - --leader-election
-            - --leader-election-lease-duration=${LEADER_ELECTION_LEASE_DURATION}
-            - --leader-election-renew-deadline=${LEADER_ELECTION_RENEW_DEADLINE}
-            - --leader-election-retry-period=${LEADER_ELECTION_RETRY_PERIOD}
-            - --v=${LOG_LEVEL}
-          env:
-          - name: ADDRESS
-            value: /var/lib/csi/sockets/pluginproxy/csi.sock
-          volumeMounts:
-          - mountPath: /var/lib/csi/sockets/pluginproxy/
-            name: socket-dir
-          resources:
-            requests:
-              memory: 50Mi
-              cpu: 10m
-        - name: snapshotter-kube-rbac-proxy
-          args:
-          - --secure-listen-address=0.0.0.0:9215
-          - --upstream=http://127.0.0.1:8215/
-          - --tls-cert-file=/etc/tls/private/tls.crt
-          - --tls-private-key-file=/etc/tls/private/tls.key
-          - --logtostderr=true
-          image: ${KUBE_RBAC_PROXY_IMAGE}
-          imagePullPolicy: IfNotPresent
-          ports:
-          # Due to hostNetwork, this port is open on a node and cannot
-          # conflict with any port used by other CSI drivers.
-          - containerPort: 9215
-            name: snapshotter-m
```

Comment:
This is also referenced in the ServiceMonitor manifest.

Reply:
Missed that, thanks.

Reply:
This is also referenced in RBAC and Services, so removing it from there as well.